oi/type_graph/TypeGraph.h

```diff
@@ -87,7 +88,7 @@ class TypeGraph {
   std::vector<std::reference_wrapper<Type>> rootTypes_;
   // Store all type objects in vectors for ownership. Order is not significant.
   std::vector<std::unique_ptr<Type>> types_;
-  NodeTracker tracker_;
+  mutable NodeTracker tracker_;
```

Comment:
This is a bit of a hack, but I remove NodeTracker from TypeGraph entirely in the next PR: #284